src/main/java/com/playground/playground/GenerateDatasets.java

```diff
@@ -34,13 +36,25 @@ public static ArrayList<ArrayList<ArrayList<Double>>> generateCircular() {
             x1.add(xp);
             y1.add(yp);
         }
+        // Add extra random points (noise)
+        for (int i = 0; i < noise * 10; i++) {
```

Comment:
WHat si the usual range of x,y if we have no noise?

Reply:
It is -10 to +10

Reply:
Do you want to reduce the noise range from 50 to 30? That should be done in the UI.